### PR TITLE
Sommelier command to toggle Wine Carrier role from a user

### LINKED ISSUE #227: BoozeBot should allow granting the Wine role
Add a command to add/remove the wine carrier role. Access for Sommelier and above.

### PR DESCRIPTION
**Pull request by cimspin**
`/make_wine_carrier @user`
Closes #227

### CODE REVIEW

File: ptn/boozebot/_metadata.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '1.4.1'
```
Comment (gcruicks) [inline ptn/boozebot/_metadata.py:1]: Revert this file please

File: ptn/boozebot/commands/Unloading.py [hunk 2]
```diff
@@ -356,3 +356,51 @@ async def wine_unloading_complete(self, ctx: SlashContext, carrier_id):
             print(f'No discord alert found for carrier, {carrier_id}. It likely ran an untracked market.')
 
         return await ctx.send(content=response)
+    @cog_ext.cog_slash(
```
Comment (gcruicks) [inline ptn/boozebot/commands/Unloading.py:359]: new line before this @cog ...

File: ptn/boozebot/commands/Unloading.py [hunk 3]
```diff
@@ -356,3 +356,51 @@ async def wine_unloading_complete(self, ctx: SlashContext, carrier_id):
...
+            # toggle on
+            print(f"{user} is not a Wine Carrier, adding the role.")
+            try:
+                await user.add_roles(role)
+                print(f"Added Wine Hauler role to {user}")
+                response = f"{user.name} now has the Wine Carrier role."
+                return await ctx.send(content=response)
+            except Exception as e:
+                print(e)
+                await ctx.send(f"Failed adding role to {user}: {e}")
```
Comment (gcruicks) [inline ptn/boozebot/commands/Unloading.py:406]: new line at the end please

File: ptn/boozebot/commands/Unloading.py [hunk 4]
```diff
@@ -356,3 +356,51 @@ async def wine_unloading_complete(self, ctx: SlashContext, carrier_id):
...
+                create_permission(server_admin_role_id(), SlashCommandPermissionType.ROLE, True),
+                create_permission(server_sommelier_role_id(), SlashCommandPermissionType.ROLE, True),
+                create_permission(server_mod_role_id(), SlashCommandPermissionType.ROLE, True),
+                create_permission(bot_guild_id(), SlashCommandPermissionType.ROLE, False),
+            ]
+        },
+    )
+    async def make_user_wine_carrier(self, ctx: SlashContext, user: discord.Member):
+        # set the target role
+        print(f"Wine Carrier role ID is {server_wine_carrier_role_id()}")
```
Comment (gcruicks) [inline ptn/boozebot/commands/Unloading.py:382]: Lets add a print of who requested to add this

**Review by gcruicks — CHANGES_REQUESTED** [review]

**Comment by cimspin** [comment]
Think all changes done now

**Review by gcruicks — APPROVED** [review]

**Pull request merged**